lib/geonames.ex

```diff
@@ -23,7 +23,8 @@ defmodule Geonames do
         token_param
       }"
 
-    case HTTPoison.get(url) do
+    # force TLS v1.2, see https://campezzi.ghost.io/httpoison-ssl-connection-closed/
+    case HTTPoison.get(url, [], ssl: [versions: [:"tlsv1.2"]]) do
```

Comment:
Should we set this as a global option on `ssl` perhaps? I wouldn't expect trouble with other places that we might be using `HTTPoison` to access third-party services, since the problematic protocol versions are deprecated and nobody should be using them. Nevertheless it would be best, if we do set this globally, to let it run on dev a while and make sure there's no uptick in errors.